lib/loga.rb

```diff
@@ -8,6 +8,7 @@
 require 'loga/rack/request'
 require 'loga/rack/request_id'
 require 'loga/railtie' if defined?(Rails)
+require 'loga/sidekiq/init' if defined?(Sidekiq)
```

Comment:
👏 I like it

Reply:
This doesn't seem to work as expected. There is some messages logged by sidekiq `... Booting Sidekiq 5.1.3 with redis options ...` so we have to change also the `Sidekiq::Logging.logger`. This means that we have to have `Loga.logger`.

Let's hook the logger in the `Loga.configure` and here just require the job logger `require 'loga/sidekiq/job_logger'`

e.g. 

```
  def self.configure(options, framework_options = {})
    if @configuration
      raise ConfigurationError, 'Loga has already been configured'
    end

    @configuration ||= Configuration.new(options, framework_options)

    if defined?(::Sidekiq) # maybe move this somewhere else
      ::Sidekiq::Logging.logger = @configuration.logger
      ::Sidekiq.configure_server do |config|
        config.options[:job_logger] = Loga::Sidekiq::JobLogger
      end
    end

    @configuration
  end
```